src/Cli/dotnet/commands/dotnet-test/TestingPlatformCommand.cs

```diff
@@ -123,15 +123,15 @@ private async Task WaitConnectionAsync(CancellationToken token)
             }
         }
 
-        private async Task<IResponse> OnRequest(IRequest request)
+        private Task<IResponse> OnRequest(IRequest request)
         {
             if (TryGetModulePath(request, out string modulePath))
```

Comment:
Here we need the try/catch/log/rethrow I think, this callback is called on another thread